contracts/core/zapper/WidoZapperVelodrome.sol

```diff
@@ -120,10 +109,10 @@ contract WidoZapperVelodrome is WidoZapperUniswapV2 {
     internal virtual override
     returns (uint256 amountOut) {
         VelodromeRouter.route[] memory routes = new VelodromeRouter.route[](1);
-            routes[0] = VelodromeRouter.route({
-            from : tokenIn,
-            to : tokenOut,
-            stable : abi.decode(extra, (bool))
+        routes[0] = VelodromeRouter.route({
+        from : tokenIn,
+        to : tokenOut,
+        stable : abi.decode(extra, (bool))
```

Comment:
indent?

Also what are you using for formatting. When I save, that whole file changes formatting. :(